scripts/eosio_uninstall.sh

```diff
@@ -1,89 +1,125 @@
 #!/usr/bin/env bash
 set -eo pipefail
+VERSION=2.1
+# User input handling
+DEP_PROCEED=false
+FULL=false
+
+function usage() {
+   printf "Usage: $0 OPTION...
+  -i DIR      Directory where eosio is installed)
+  -y          Noninteractive mode (answers yes to every prompt)
+  -f          Removal of data directory (be absolutely sure you want to delete it before using this!)
```

Comment:
Please update the README as well to reflect these changes to the uninstall script.